src/main/kotlin/com/terraformation/backend/accelerator/api/AcceleratorOrganizationsController.kt

```diff
@@ -61,8 +61,12 @@ class AcceleratorOrganizationsController(
 
     return ListAcceleratorOrganizationsResponsePayload(
         organizations.map { (organization, projects) ->
+          val contacts = userStore.getTerraformationContactUsers(organization.id)
           AcceleratorOrganizationPayload(
-              organization, projects, userStore.getTerraformationContactUser(organization.id))
+              organization,
+              projects,
+              contacts,
```

Comment:
I don't think the trailing comma here is needed since the arguments will all fit on one line.